content/oci/oci.go

```diff
@@ -52,12 +53,26 @@ type Store struct {
 	//     to manually call SaveIndex() when needed.
 	//   - Default value: true.
 	AutoSaveIndex bool
-	root          string
-	indexPath     string
-	index         *ocispec.Index
-	storage       *Storage
-	tagResolver   *resolver.Memory
-	graph         *graph.Memory
+
+	// AutoGC controls if the OCI store will automatically clean newly produced
+	// dangling (unreferenced) blobs during Delete() operation. For example the
+	// blobs whose manifests have been deleted. Manifests in the index will not
```

Comment:
You may want to mention `index.json` instead of just `index`.

Reply:
changed to `index.json`.